From 80f258b9d188b53ff3ab8953842c3a862178b2fc Mon Sep 17 00:00:00 2001
From: Josua Mayer <josua@solid-run.com>
Date: Thu, 12 Sep 2024 14:58:59 +0300
Subject: [PATCH] board: solidrun: imx8dxl-sr-som: update default fdtfile to
 production

All SoMs before v2.1, and all carriers before v2.0 were prototypes.
Update default fdtfile for production version (1.1 equivalent 2.0 in
software).

CONFIG_DEFAULT_FDT_FILE requires quotation marks for string value,
otherwise make *_defconfig ignores the value.
Add quotation marks.

Signed-off-by: Josua Mayer <josua@solid-run.com>
---
 configs/imx8dxl_v2x_defconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configs/imx8dxl_v2x_defconfig b/configs/imx8dxl_v2x_defconfig
index f1e7bd50c6a..1515b174813 100644
--- a/configs/imx8dxl_v2x_defconfig
+++ b/configs/imx8dxl_v2x_defconfig
@@ -193,4 +193,4 @@ CONFIG_BOOTCOMMAND="run distro_bootcmd"
 CONFIG_SUPPORT_RAW_INITRD=y
 
 # default fdtfile name
-CONFIG_DEFAULT_FDT_FILE=imx8dxl-v2x.dtb
+CONFIG_DEFAULT_FDT_FILE="imx8dxl-v2x-v11-som-v21.dtb"
-- 
2.43.0

